modules/cats/src/main/scala/pureconfig/module/Cats.scala

```diff
@@ -1,15 +0,0 @@
-package pureconfig.module
-
-import pureconfig.error.FailureReason
-
-object Cats {
```

Comment:
It duplicates `EmptyTraversableFound` from package object. Does it make sense to simply remove it or it's better to mark deprecated since , let's say,  `0.11.0`?

Reply:
Well spotted! Since it's not being used anywhere in the code it should be safe to delete.